lotus-soup/testkit/role_miner.go

```diff
@@ -245,6 +250,20 @@ func PrepareMiner(t *TestEnvironment) (*LotusMiner, error) {
 	// 	panic(err)
 	// }
 
+	// set seal delay to lower value than 1 hour
+	err = n.MinerApi.SectorSetSealDelay(ctx, sealDelay)
+	if err != nil {
+		return nil, err
+	}
+
+	// print out the admin auth token
+	token, err := n.MinerApi.AuthNew(ctx, apistruct.AllPermissions)
+	if err != nil {
+		return nil, err
+	}
+
+	t.RecordMessage("Auth token: %s", string(token))
```

Comment:
Now that we no longer start permissioned APIs, this is not 100% necessary, but probably worth to have this token generated and printed out just in case.